go-controller/hack/update-codegen.sh

```
@@ -73,8 +77,33 @@ for crd in ${crds}; do
     github.com/ovn-kubernetes/ovn-kubernetes/go-controller/pkg/crd/$crd/${api_version} \
     "$@"
 
+  echo "Generating openapi for $crd ($api_version)"
+  # this can be moved to apis/ folder when the cmd becomes auto-generated
+  new_report="$(mktemp -t "$(basename "$0").api_violations.XXXXXX")"
+  openapi-gen \
+    --output-file zz_generated.openapi.go \
+    --output-dir "${SCRIPT_ROOT}"/pkg/crd/$crd/${api_version}/openapi \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)